systemtest/src/test/java/io/strimzi/systemtest/operators/LeaderElectionIsolatedST.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.operators;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.fabric8.kubernetes.api.model.PodBuilder;
+import io.fabric8.kubernetes.api.model.coordination.v1.Lease;
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.annotations.IsolatedSuite;
+import io.strimzi.systemtest.storage.TestStorage;
+import io.strimzi.systemtest.utils.StUtils;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import io.strimzi.test.annotations.IsolatedTest;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.strimzi.systemtest.Constants.INFRA_NAMESPACE;
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static io.strimzi.systemtest.resources.ResourceManager.kubeClient;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+
+/**
+ * Suite for testing Leader Election feature <br>
+ *
+ * The feature allows users to run Cluster operator in more than one replica <br>
+ *
+ * There will be always one leader, other replicas will stay in "standby" mode <br>
+ *
+ * The whole procedure of deploying CO with Leader Election enabled and many more is described in
+ *
+ * <a href="https://strimzi.io/docs/operators/in-development/configuring.html#assembly-using-multiple-cluster-operator-replicas-str">the documentation</a>
```

Comment:
Hmmm, I guess that referring to doc as a specific URL is not great because it could change anytime... :) Wouldn't be better to reference just `https://strimzi.io/docs/operators/in-development/configuring.html`?

Reply:
Makes sense

Reply:
I would keep it. In the worst case, if the anchor is removed or renamed, you will still get to https://strimzi.io/docs/operators/in-development/configuring.html